src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,72 @@
+package lotto.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lotto.exception.DuplicateLottoNumberException;
+import lotto.exception.InvalidRangeLottoNumberException;
+import lotto.exception.InvalidSizeLottoNumberException;
+
+public class Lotto {
+    private static final int LOTTO_SIZE = 6;
+    private static final int MIN_RANGE = 1;
+    private static final int MAX_RANGE = 45;
+
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        numbers = sorted(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        validateSize(numbers);
+        validateRange(numbers);
+        validateDuplicate(numbers);
+    }
+
+    private void validateSize(List<Integer> numbers) {
+        if (numbers.size() != LOTTO_SIZE) {
+            throw new InvalidSizeLottoNumberException();
+        }
+    }
+
+    private void validateRange(List<Integer> numbers) {
+        numbers.forEach(this::validateRange);
+    }
+
+    private void validateRange(Integer number) {
+        if (!(MIN_RANGE <= number && number <= MAX_RANGE)) {
+            throw new InvalidRangeLottoNumberException();
+        }
+    }
```

Comment:
메서드 명이 중복되고 있는데요, 같은 메서드를 오버로딩 하는 것 역시 하나의 방법이지만 메서드의 역할을 더 잘 나타내기 위해선 디테일한 부분을 나타낼 수있도록 서로 다른 메서드명을 주는 것도 좋을 것 같아요.
```suggestion
    private void validateNumberList(List<Integer> numbers) {
        numbers.forEach(this::validateNumberInRange);
    }

    private void validateNumberInRange(Integer number) {
        if (!(MIN_RANGE <= number && number <= MAX_RANGE)) {
            throw new InvalidRangeLottoNumberException();
        }
    }
```